### PR TITLE
Requirements

### PR DESCRIPTION
**Pull request by binhussen**
finished up all requirements

### CODE REVIEW

**Review by Reem-lab — CHANGES_REQUESTED** [review]
## Hi @binhussen , 👋🏻👋🏻

Good job so far!🔥♻💯🚀
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
 - Good GitHub flow ✅✅


## Required Changes ♻️
_Check the comments under the review._

- [ Required ] I think you forget to add linters files for HTML and CSS, please Add linters file to this project, you can check it in the requirements linters for HTML and CSS♻
- [ Required ] please your Pull Request should have a summary describing introduced changes in more detail.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Reem-lab — APPROVED** [review]

File: .github/workflows/linters.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup Lighthouse
+        run: npm install -g @lhci/cli@0.7.x
+      - name: Lighthouse Report
+        run: lhci autorun --upload.target=temporary-public-storage --collect.staticDistDir=.
```
Comment (Reem-lab) [inline .github/workflows/linters.yml:20]: - [ ] Kindly add the `hitrc` and `stylelint` files to the project so you can check the linters through it.

**Review by Reem-lab — CHANGES_REQUESTED** [review]

**Comment by binhussen** [comment]
## Hello Microverse
In this PR I created a Hello Microverse Project with In the below checklists
### CheckList
- Create new GitHub repo and branch.
- Add a .gitignore file.
- SetUp linter for HTML and CSS.
- Add a descriptive README file to your project.
- Add .html file, .css file, and test.MD file with separate commit.
- Ignore the test.MD file by git.

**Review by ShahierNashaat — APPROVED** [review]
Hi @binhussen , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Approved :ok_hand: :ok_hand: :ok_hand:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**